### PR TITLE
Daily checker update (14-12-20)

### PR DESCRIPTION
**Pull request by benthorner**
https://trello.com/c/hFgWD3nP/705-checker-action-mini-batch-update-december

Please see the commits for more details.

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

**Comment by benthorner** [comment]
@hannako as discussed, I've updated the last commit with a notification. Could you take another look?

File: app/lib/brexit_checker/notifications.yaml [hunk 1]
```diff
@@ -68,3 +68,13 @@ notifications:
...
+  - uuid: a4294202-72f5-4752-8e8a-5c18bb8afa83
+    type: content_change
+    action_id: S005
+    date: '2020-12-14'
+    note: The action has been updated to apply in general to EU students studying in the UK, not just Erasmus students.
+  - uuid: 5e29982f-a6e0-42e4-bea2-f44486079f59
+    type: content_change
+    action_id: S026
+    date: '2020-12-14'
+    note: Updated action so that it covers all students who are planning to study in the EU, not just Erasmus students continuing their studies in EU.
```
Comment (hannako) [inline app/lib/brexit_checker/notifications.yaml:80]: Following chat with @joeharrison91 please can we send this notification to the smaller subset of users that meet criteria: `nationality-uk` and `studying-uk`

**Review by hannako — CHANGES_REQUESTED** [review]

File: app/lib/brexit_checker/notifications.yaml (hunk 1 above)
Comment (benthorner) [reply app/lib/brexit_checker/notifications.yaml:80]: Sure, I've updated with that, and the commit message tries to justify it 👍.

**Review by hannako — APPROVED** [review]
LGTM

**Pull request merged**